docs/commands/lint.md

```diff
@@ -2,239 +2,378 @@
+##### Method 2: Pass entrypoints via the config file
 
-#### Pass entrypoints via configuration file
+The `apis` section in the Redocly configuration file (`redocly.yaml`) lets you configure one or more APIs. Each file path is specified (using the `root` property), but files are identified by their name and version in the format `name@version`.
 
-Instead of full paths, you can use names listed in the `apis` section of your Redocly configuration file as entrypoints.
+`@version` is optional, and when not provided, Redocly apps interpret it as `latest` by default. Every `name@version` combination listed in the `apis` section must be unique.
 
-```bash Command
-openapi lint main
-```
+You can pass `name` or `name@version` in commands like `lint` and `stats` to act only on those APIs. Below is an example `apis` section showing three files identified using `main`, `production` and `sandbox`:
 
 ```yaml Configuration file
 apis:
   main:
-    root: ./openapi/definition.json
+    root: ./openapi.yaml
+    labels:
+      - main
+    lint: []
+    features.openapi: []
+  production:
+    root: ./production.yaml
+    labels:
+      - production
+    lint: []
+    features.openapi: []
+  sandbox:
+    root: ./sandbox.yaml
+    labels:
+      - sandbox
+    lint: []
+    features.openapi: []
 ```
 
-In this case, after resolving the path behind the `main` name (see the `Configuration file` tab), `lint` will validate the `definition.json` file. The presence of the Redocly configuration file is mandatory.
+ In this example, we only want to validate files under `main`:
 
-#### Empty entrypoints
+```bash Command
+redocly lint main
+```
 
-You can omit entrypoints completely when executing the `lint` command.
+##### Method 3: Pass no entrypoints
 
-```bash Command
-openapi lint
+Running `lint` without specifying a particular API definition file checks all definition files in your project directory.
+
+```bash
+redocly lint
 ```
 
-```yaml Configuration file
-apis:
-  main:
-    root: ./openapi/definition.json
-  production:
-    root: ./openapi/production.yaml
-  sandbox:
-    root: ./openapi/sandbox.yaml
+:::warning Important
+If you run `redocly lint` without entrypoints, and your project has no configuration files, you'll get an error.
+:::
+
+### How to reference a custom config file
+
+By default, Redocly CLI looks for the [Redocly configuration file](/docs/cli/configuration/configuration-file.mdx) in the current working or project directory. Using the optional `--config` argument provides an alternative path to a different configuration file if one exists.
+
+```bash
+redocly lint --config=./another/directory/anotherconfig.yaml
 ```
 
-In this case, if no API definitions are specified, `lint` validates all entrypoints listed under `apis` in your Redocly configuration file. The presence of the configuration file is mandatory.
+### How to activate rule sets
```

Comment:
```suggestion
### How to activate rule configurations
```

Reply:
This is my mistake. I think I used this term, _rule sets_, to explain this functionality to you at some point. I also used it in other places in the docs. Later, Adam warned me not to use that term anymore, so I removed it but forgot to tell you. Sorry about that. I think we must use "rule configurations" instead, or just "configuration" when it's clear from the context we're talking about rules.